app/src/main/res/layout/sheet_blood_sugar_entry.xml

```diff
@@ -0,0 +1,246 @@
+            app:layout_constraintStart_toEndOf="@+id/bloodsugarentry_day_month_separator"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:ignore="UnusedAttribute" />
+
+          <TextView
+            android:id="@+id/bloodsugarentry_month_label"
+            style="@style/Clinic.BloodSugarInputLabel"
+            android:gravity="center_horizontal"
+            android:labelFor="@id/monthEditText"
+            android:text="@string/bloodsugarentry_month"
+            android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Grey1"
+            app:layout_constraintEnd_toEndOf="@+id/monthEditText"
+            app:layout_constraintStart_toStartOf="@+id/monthEditText"
+            app:layout_constraintTop_toBottomOf="@+id/monthEditText" />
+
+          <TextView
+            android:id="@+id/bloodsugarentry_month_year_separator"
+            style="@style/Clinic.V2.BloodSugarInputSeparator"
+            app:layout_constraintBottom_toBottomOf="@+id/monthEditText"
+            app:layout_constraintEnd_toStartOf="@+id/yearEditText"
+            app:layout_constraintStart_toEndOf="@+id/monthEditText"
+            app:layout_constraintTop_toTopOf="@+id/monthEditText" />
+
+          <EditText
+            android:id="@+id/yearEditText"
+            style="@style/Clinic.V2.BloodSugarInput.Date"
+            android:importantForAutofill="no"
+            android:theme="@style/Clinic.V2.BloodSugarInputTheme"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/bloodsugarentry_month_year_separator"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:ignore="UnusedAttribute" />
+
+          <TextView
+            android:id="@+id/bloodsugarentry_year_label"
+            style="@style/Clinic.BloodSugarInputLabel"
+            android:gravity="center_horizontal"
+            android:labelFor="@id/yearEditText"
+            android:text="@string/bloodsugarentry_year"
+            android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Grey1"
+            app:layout_constraintEnd_toEndOf="@+id/yearEditText"
+            app:layout_constraintStart_toStartOf="@+id/yearEditText"
+            app:layout_constraintTop_toBottomOf="@+id/yearEditText" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <TextView
+          android:id="@+id/dateErrorTextView"
+          style="@style/Clinic.V2.TextAppearance.TextInputLayoutError"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="@dimen/spacing_8"
+          android:gravity="center_horizontal"
+          tools:text="@string/bloodsugarentry_error_date_is_in_future" />
+      </LinearLayout>
+    </FrameLayout>
+  </org.simple.clinic.widgets.ViewFlipperWithLayoutEditorPreview>
+</org.simple.clinic.widgets.LinearLayoutWithPreImeKeyEventListener>
```

Comment:
Yes, there are few minor changes in terms of spacing and adding an icon to the title.

Reply:
It would have been easier to review had we split this commit into copying the code and making changes for blood sugar. It's fine now. Maybe try this next time.